### ISSUE TITLE
[Bug]: Accessibility tests fail because there is no label specified in checkbox of Card when toggles property is used in sp-card.

### ISSUE THREAD

**Issue by vaishnavijha** [adobe/spectrum-web-components#2224]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-card

### Expected behavior

Accessibility tests should pass for [Card component](https://git.corp.adobe.com/Horizon/hz/blob/d42400faf3fcb0d8f14c48f77f1657d6d67f7ca7/apps/project-x/web/src/components/card/Card.ts#L66) in my code using toggles property which is built on top of sp-card from web spectrum .

### Actual behavior

When I use toggles property in my Card component which is built on sp-card , my accessibility tests fail .

This might be because of implementation of toggles feature for spectrum web card component(sp-card).This component uses `sp-checkbox` component to select a card without label [sp-card](https://github.com/adobe/spectrum-web-components/blob/341f4932087487be47bde355d1b0894886ed44ad/packages/card/src/Card.ts#L304).

Also [sp-checkbox](https://github.com/adobe/spectrum-web-components/blob/2429bb461782c54c95c3f46bede6fd28254fd4e4/packages/checkbox/src/Checkbox.ts#L114) is being used without any label, this label slot remains empty and hence this is throwing accessibility error.

### Screenshots

![image](https://user-images.githubusercontent.com/39922632/163111642-ceafb5c7-e4b5-4e54-9d26-36e77d1e5e6d.png)

### What browsers are you seeing the problem in?

Firefox, Chrome, Safari, Microsoft Edge

### How can we reproduce this issue?

1. Take a pull from project-x-operations-store branch https://git.corp.adobe.com/Horizon/hz/tree/project-x-operations-demo
2. Build your project with these commands 

```
bin/setup.sh (from hz root)
rush update (within project-x/web)

```
3. Go to apps/project-x/web/src/components/card/Card.spec.ts
4. Add these lines of code which are removed for now just to get our tests passed 
```
it("loads Card accessibly", async () => {
        const el = await fixture<Card>(html` <x-card></x-card> `);
        await elementUpdated(el);
        await expect(el).to.be.accessible();
    });
```
<img width="871" alt="image" src="https://user-images.githubusercontent.com/39922632/163112794-86dbf72b-c579-49d1-80b4-d02dafee39d6.png">

5.run this command on terminal `rushx test:watch --files "dist/src/components/card/Card.spec.js"`
6.You will see this accessibility test always fails with missing label message .

### Logs taken while reproducing problem

![image](https://user-images.githubusercontent.com/39922632/163113623-8f066d66-3479-4749-91d1-574912142dbc.png)

**Comment by Westbrook** [comment]
@vaishnavijha can you share the expected shadow DOM of your `<x-card></x-card>` test?

Also, I do not see how `toggles` is getting applied in order to activate this code path.

Nor do I see how _any_ label text would be available to point to in that test case, so even were we to make the correct associations internally the test would still fail for lack of content.

**Comment by vaishnavijha** [comment]
<img width="1147" alt="image" src="https://user-images.githubusercontent.com/39922632/163312289-aadf0679-58cf-4265-a2e2-1c46603cee4b.png">
This is the way we were passing content to x-card in our other tests , I was under the impression that even if I just call x-card , accessibility should get passed irrespective of content inside , Thanks for pointing this out @Westbrook ,Feel free to close this issue now.

**Comment by Westbrook** [comment]
Unable to recreate failure in repo to support further changes or regression prevention, so closing as per OP.
